pack.ps1

```diff
@@ -68,9 +72,12 @@ function UpdateTaskManifests($workingDirectory, $version, $envName) {
         Write-Host "Updating version to $version in $taskManifestFile..."
         $task = ConvertFrom-JSON -InputObject (Get-Content $taskManifestFile -Raw)
         $netVersion = [System.Version]::Parse($version)
-        $task.version.Major = $netVersion.Major
-        $task.version.Minor = $netVersion.Minor
-        $task.version.Patch = $netVersion.Build
+
+        if ($task.version.Major -ne 3) {
```

Comment:
I feel like this could go either way, depending on what we decide to keep shipping after the next major version update, but I've implemented this suggestion.